src/semantic-router/pkg/config/config.go

```diff
@@ -336,6 +336,36 @@ func LoadConfig(configPath string) (*RouterConfig, error) {
 	return config, nil
 }
 
+// BoolPtr returns a pointer to a bool value (helper for tests and config)
```

Comment:
I noticed that this function was only used in the tests. Should it be moved into the tests section instead? The functional code doesn't use it. However, I don't think it's a major issue. It can be improved later.